src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,25 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+import mate.academy.model.User;
+
+public class AuthenticationService extends UserService {
```

Comment:
add UserService as class field
```suggestion
public class AuthenticationService {
```